### DISCUSSION TITLE
April Fools Addons

### DISCUSSION THREAD

**Discussion by TheColaber** [ScratchAddons/ScratchAddons#1915] [Help]
So april fools day is coming, and we've decided what better way to fool ppl then to just slap a version two point o on there and add some fake addons. We need some ideas of fake addons to include...

Here is the list I have right now (of which i am "implementing" as well):

- [x] project collaboration
- [x] private messaging
- [x] private studios
- [x] Markdown in comments
- [x] A button to invite all followers to a studio
- [x] set project share time
- [x] font customization
- [ ] your ideas here...

Also we can have other cool things like the tabs in the popup become cat tabs, the settings popup is boarded with tape, etc.

**Comment by LES-11** [comment]
interesting 
while yes we must fool them somehow (obviously), getting users hopes up about an addon that does not exist might not be the best way to please people and keep them as SA users

this is just my opinion 
but I agree we must fool them somehow

**Reply by TheColaber** [reply]
I don't care if we loose 10,000 users because of this. It won't happen, and it's just a fun joke. It's what people want hehe

**Reply by LES-11** [reply]
id clear it with W_L first...

**Reply by TheColaber** [reply]
He has ;) https://discord.com/channels/751206349614088204/819307061493760031/821383901641572462

**Comment by mxmou** [comment]
Add [this](https://github.com/ScratchAddons/ScratchAddons/issues/1646#issuecomment-783448997) (maybe not mandatory, but enabled by default).

**Reply by Explosion-Scratch** [reply]
Comic sans as default >=)

**Reply by ghost** [reply]
Or maybe Wingdings or Webdings?

**Reply by InsanityFromScratch** [reply]
> * [x]  font customization

Font customization would actually make a good addon.

**Reply by nexensys** [reply]
How come people say this now, but when I made the addon it got kicked?

**Comment by retronbv** [comment]
Just saying: people have access to the github repo, if they check often they will figure it out

**Reply by TheColaber** [reply]
Same goes with scratch... it doesn't really matter...

**Comment by Explosion-Scratch** [comment]
I'm thinking for PM'ing we add a button next to reply in messaging popup, which when clicked opens a textbox, lets you type, then when clicking submit takes you to [this awesome resource](https://www.youtube.com/watch?v=dQw4w9WgXcQ)

**Comment by Explosion-Scratch** [comment]
Project collaboration we can implement fake cursors moving around using random movement. (Think random velocities away from the edges of the screen)

**Reply by retronbv** [reply]
🙌 holy explosion with the holy ideas

**Comment by scratchusernamemrtbts** [comment]
how about tricking that griffpatch followed them
and then another day say that griffpatch unfollowed them

**Reply by retronbv** [reply]
lmao great idea

**Reply by FunctionalMetatable** [reply]
or maybe an ST member aswell like paddle2see

**Comment by TheColaber** [comment]
Please note that we don't want to actually interact with scratch, and only the extension.

**Reply by mxmou** [reply]
So joke addons can't be enabled by default?

**Reply by TheColaber** [reply]
no... they will all do nothing... W_L is thinking of making easter eggs within scratch if the user opts in like adding some kind of text in their bio... then it can make it rain dangos or something.

**Reply by mxmou** [reply]
Why did you upvote my suggestion for mandatory Sora then?

**Reply by TheColaber** [reply]
Which issue? This is about fooling people as far as I know...

**Reply by TheColaber** [reply]
Oh I see, one of your posts above... i upvoted it to be one of the fake addons... they don't actually do anything tho...

**Comment by ghost** [comment]
I like the idea of fooling people who use it like me XD but I do wish you could really add those.

**Comment by gazalied** [comment]
Viewing passwords on people's profile

**Reply by Endermaster3690** [reply]
perfect prank

**Comment by ghost** [comment]
how about an addon that saysit lets you select any animal for the hats not just cat blocks XD

**Comment by CST1229** [comment]
Dislike button. (EDIT: All it does is display a greyed out button.)

**Reply by ghost** [reply]
XD

**Comment by Basic8** [comment]
AFD is only a day away, but here's an idea: Add a new topic button in announcements and then it will take you to the rickroll video
